samples/drivers/adc/boards/atsaml21_xpro.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2021 Argentum Systems Ltd.
+ */
+
+/ {
+	zephyr,user {
+		/* EXT-1, pin 3 ADC(+) */
+		io-channels = <&adc 13>;
+	};
+};
+
+&pinctrl {
+	adc_default: adc_default {
+		group1 {
+			pinmux = <PB5B_ADC_AIN13>;
+		};
+	};
+};
+
+&adc {
+	pinctrl-0 = <&adc_default>;
+	pinctrl-names = "default";
+};
```

Comment:
you need to provide channel configuration in DT. Otherwise this sample won't work, see https://github.com/zephyrproject-rtos/zephyr/pull/46110/commits/94d703cd48ae97a1c7de884937e82f7ed1a233c7

Reply:
Looks like the other SAM0 samples are broken too... `zephyr,input-positive` is missing... I'll see about a fix separately, but I can't test it.